test/project_test.py

```diff
@@ -124,7 +124,9 @@
     },
 
     "animation": {
-        "typename": "bibliopixel.animation.tests.StripChannelTest"
+        "typename": "bibliopixel.animation.tests.StripChannelTest",
+        "name": "test name",
```

Comment:
For remote I'm going to have to enforce that names be alpha-num only and not have spaces, just FYI.